project/dexter-monitor/public/ctrl/userStatusCtrl.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2016 Samsung Electronics, Inc.,
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * * Redistributions of source code must retain the above copyright notice, this
+ *   list of conditions and the following disclaimer.
+ *
+ * * Redistributions in binary form must reproduce the above copyright notice,
+ *   this list of conditions and the following disclaimer in the documentation
+ *   and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+"use strict";
+
+monitorApp.controller("UserStatusCtrl", function($scope, $http, $log, UserService, uiGridConstants) {
+
+    const columnDefs = [
+        {field:'groupName',                 displayName:'부서명',      width: 215,
+            cellClass: 'grid-align',    headerTooltip: 'Group name'},
+        {field:'allDeveloperCount',         displayName:'총 개발자 수',  width: 135,
+            cellClass: 'grid-align',    headerTooltip: 'Number of developers',
+            aggregationType: uiGridConstants.aggregationTypes.sum},
+        {field:'targetDeveloperCount',      displayName:'설치 대상',    width: 135,
+            cellClass: 'grid-align',    headerTooltip: 'Number of developers who should install Dexter',
+            aggregationType: uiGridConstants.aggregationTypes.sum},
+        {field:'installedDeveloperCount',   displayName:'설치 완료',    width: 135,
+            cellClass: 'grid-align',    headerTooltip: 'Number of developers who installed Dexter',
+            aggregationType: uiGridConstants.aggregationTypes.sum},
+        {field:'installationRate',          displayName:'설치율(%)',   width: 135,
```

Comment:
It is an outdated file.
Already changed to `field:'installationRatio'` and `displayName: 'Installation ratio (%)'`.